exe/appmap-agent-status

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'appmap'
+require 'appmap/command/agent_setup/status'
+
+@options = {:config_file => AppMap::DEFAULT_CONFIG_FILE_PATH}
+
+OptionParser.new do |parser|
+  parser.banner = 'Usage: bundle exec exe/appmap-agent-status [options]'
```

Comment:
Let's explain the usage simply as `Usage: appmap-agent-status [options]`. Bundler is optional and the user may be running the command from a binstub or executable file installed by Rubygems.

Reply:
Also, `exe/appmap-agent-status` is only correct when running the command from the gem root directory; but the user will normally be running it from their own project root directory. Since the `appmap` gem is installed, `bundle exec appmap-agent-status` should work.